core/server/proxy/src/main/java/alluxio/proxy/s3/S3ErrorCode.java

```diff
@@ -23,8 +23,10 @@
    */
   public static final class Name {
     public static final String BUCKET_ALREADY_EXISTS = "BucketAlreadyExists";
+    public static final String BUCKET_NOT_EMPTY = "BucketNotEmpty";
     public static final String INTERNAL_ERROR = "InternalError";
     public static final String INVALID_BUCKET_NAME = "InvalidBucketName";
+    public static final String NO_SUCH_BUCKET = "BucketNotExists";
```

Comment:
I'm following the spec here: http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html

It's called `NoSuchBucket` in s3 error code docs.